### PR TITLE
[Binance Indonesia SDK] Add Transactions Overview Flow

### PR DESCRIPTION
**Pull request by rebekahlow-jy**
## Approach
- Add withdrawal requests into requests tab
- Localise transactions overview for Indo

## Screenshots
Screenshot 2019-08-01 at 11 18 17 AM
Screenshot 2019-08-01 at 11 18 22 AM

### CODE REVIEW

File: JavaScript/src/components/screens/TransactionsOverview/components/TransactionsTable/TransactionsTable.js [hunk 1]
```diff
@@ -19,10 +19,7 @@ export default class TransactionsTable extends Component {
 
     return (
       <StickyPanel showBrand>
-        { params.goBackPreviousModule ?
```
Comment (lfkeong) [inline JavaScript/src/components/screens/TransactionsOverview/components/TransactionsTable/TransactionsTable.js]: how come you remove this?

File: JavaScript/src/wrappers/helpers/NetworkClient.js [hunk 2]
```diff
@@ -334,4 +334,8 @@ export default class Xfers {
   getPayoutRequests() {
     return this.api.send('GET', 'payouts');
   }
+
+  getWithdrawals(params) {
+    return this.api.send('GET', 'user/bank_account/withdrawal_requests?start_date=' + params.start_date + '&end_date=' + params.end_date);
```
Comment (lfkeong) [inline JavaScript/src/wrappers/helpers/NetworkClient.js]: Should filter by status query

File: JavaScript/src/components/screens/TransactionsOverview/components/TransactionsTable/TransactionsTable.js (hunk 1 above)
Comment (rebekahlow-jy) [reply JavaScript/src/components/screens/TransactionsOverview/components/TransactionsTable/TransactionsTable.js]: Removed this, cos it doesn't work haha

File: JavaScript/src/wrappers/helpers/NetworkClient.js (hunk 2 above)
Comment (rebekahlow-jy) [reply JavaScript/src/wrappers/helpers/NetworkClient.js]: filtered in frontend using selectors!

**Pull request merged**